Aptoide/aptoide-client-v8
# [APP-1309] Fix design

**What does this PR do?**

This PR aims at implementing design fixes in v10 for home, editorial and appview.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] BundlesScreen.kt

**How should this be manually tested?**

  Open the app and check the design. This will be confirmed with Silvana, our designer.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-1309](https://aptoide.atlassian.net/browse/APP-1309)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
